src/axom/quest/examples/shaping_driver.cpp

```diff
@@ -147,6 +148,12 @@ struct Input
       ->check(axom::CLI::NonNegativeNumber)
       ->capture_default_str();
 
+    app.add_option("-e,--percent-error", percentError)
+      ->description(
+        "Percent error in calculating curve refinement and revolved volume")
+      ->check(axom::CLI::PositiveNumber)
+      ->capture_default_str();
```

Comment:
Since this is a percent, should it be supplied as a number between 0 and 100 rather than 0 and 1?
e.g. `--percent-error 1` for 1% instead of `--percent-error 0.01` ?

An alternate name for a number between 0 and 1 might be `error-threshold`.

Note: I'm ok with it as is, but thought I'd float the idea

Reply:
I can change the percent to [0,100] - it's probably more intuitive.

Reply:
I agree with that choice.

Reply:
I made the --percent-error value take a double value in (0,100) and I made changes internally to deal with the change.